puppet/conveyancer_frontend/templates/run.sh.erb

```diff
@@ -9,5 +9,6 @@ pip3 install -r requirements.txt
 port="${B_GUNICORN_PORT:-8000}"
 host="${B_GUNICORN_HOST:-0.0.0.0}"
 
+export CASE_API_BASE_HOST="<%=@case_api_base_host%>"
```

Comment:
Could you add this to the service.systemd.erb. There's an Environment part where you can define ENV for that process.